test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -1025,6 +1027,40 @@ def test_qconfig_dict_object_type_module(self):
         self.assertTrue(isinstance(mq[1], nn.Hardswish))
         self.assertTrue(isinstance(mq[2], nnq.Conv2d))
 
+    def test_qconfig_dict_object_type_function(self):
+        """
+        Verifies that the 'object_type' option of qconfig_dict works
+        on function types.
+        """
+        class M(nn.Module):
+            def forward(self, x):
+                x = x + x
+                x = x * x
+                return x
+
+        m = M()
+        # TODO(future PR): also implement global qconfig being None
+        # and individual functions having qconfigs
+        qconfig_dict = {
+            '': torch.quantization.default_qconfig,
+            'object_type': [
```

Comment:
in fx, 'object_type' can be module_type, torch function/torch ops, and methods (call_method node in fx), do we plan to support modules and call_methods as well? currently we use strings for call_method, e.g. "add", which probably means `torch.Tensor.add` here, maybe we want to standardize this api as well. I feel it might be better to use `torch.Tensor.add` as the standard format, and have a map from string to it in fx, what do you think?

Reply:
looks like the others might also be supported already, can we add two more tests, one for module, one for method (torch.Tensor.add)?

Reply:
sure, the module tests are added in https://github.com/pytorch/pytorch/pull/69719 (earlier in this stack), we can add the method test to this PR

Reply:
let me cover methods in a separate PR

Reply:
> I feel it might be better to use torch.Tensor.add as the standard format, and have a map from string to it in fx, what do you think?

yes, after looking into methods in DBR, I agree with that.